### PR TITLE
Error on trailing path separator in copy command

### LINKED ISSUE #201: acbuild copy does not consider targets ending in / as directories
It seems that "acbuild copy" does not consider a target path as a directory when it ends with / and thus the following two exhibit different behavior:

```
acbuild copy file /path/to/
acbuild copy file /path/to/file
```

The first one acts just like `acbuild copy file /path/to`

It seems that the problem happens because of this in lib/copy.go:CopyToTarget():

```
target := path.Join(a.CurrentACIPath, aci.RootfsDir, to)
```

since Join() removes any trailing / which then causes the subsequent path.Split() to return the last directory as a file.

### PR DESCRIPTION
**Pull request by sanmai-NL**
Resolves #201. See https://github.com/containers/build/issues/201#issuecomment-215243446.

### CODE REVIEW

File: acbuild/copy.go [hunk 1]
```diff
@@ -45,6 +46,13 @@ func runCopy(cmd *cobra.Command, args []string) (exit int) {
 		stderr("Copying host:%s to aci:%s", args[0], args[1])
 	}
 
+	if args[1][len(args[1])-1] == os.PathSeparator {
+		stderr(`There is a trailing path separator in the second 
+			operand. Please review the differences between 
+			this command and copy-to-dir. `)
+		return 1
+	}
+
```
Comment (dgonyeo) [inline acbuild/copy.go]: I think this will panic if `args[1]` is `""`, can you fix that?

Comment (sanmai-NL) [inline acbuild/copy.go]: Solved.

**Comment by dgonyeo** [comment]
If you can squash the commits I'll merge

**Comment by sanmai-NL** [comment]
@dgonyeo: done.

**Pull request merged**